scripts/completion/dvc.bash

```diff
@@ -71,6 +71,10 @@ _dvc_version=''
 #       x="hello"
 #       ${!x} ->  ${hello} ->  "world"
 #
+replace_hyphen () {
```

Comment:
Not quite sure how bash is handling namespacing for completion scripts, so a bit worried about it colliding with something and so wouldn't place this function like that, but would just define it inside of `_dvc` function as a nested function. That way there is no chance of it colliding with anything.